tools/CouchDB_Import_Instruments.php

```diff
@@ -78,83 +109,121 @@ function UpdateDataDicts($Instruments)
+                    'Meta'           => ['DataDict' => true],
+                    'DataDictionary' => [$instrument => $Dict],
+                ]
             );
         }
     }
 
+    /**
+     * Generates SQL statement
+     *
+     * @param string $tablename Name of the database table
+     *
+     * @return string
+     */
     function generateDocumentSQL(string $tablename) : string
     {
-        $select = "SELECT 
-                        c.PSCID, 
-                        s.Visit_label, 
-                        f.Administration, 
-                        f.Data_entry, 
-                        f.Validity,
-                        f.CommentID, 
-                        CASE WHEN EXISTS (SELECT 'x' FROM conflicts_unresolved cu WHERE f.CommentID=cu.CommentId1 OR f.CommentID=cu.CommentId2) THEN 'Y' ELSE 'N' END AS Conflicts_Exist, 
-                        CASE ddef.Data_entry='Complete' WHEN 1 THEN 'Y' WHEN NULL THEN 'Y' ELSE 'N' END AS DDE_Complete ";
-        $from = "FROM 
-                        flag f 
-                        JOIN session s ON (s.ID=f.SessionID) 
-                        JOIN candidate c ON (c.CandID=s.CandID) 
-                        LEFT JOIN flag ddef ON (ddef.CommentID=CONCAT('DDE_', f.CommentID)) ";
-        $where = "WHERE 
-                        f.CommentID NOT LIKE 'DDE%' 
-                        AND f.Test_name=:inst
-                        AND s.Active='Y' AND c.Active='Y'";
+        $select = <<<'QUERY'
+SELECT
+    c.PSCID,
+    s.Visit_label,
+    f.Administration,
+    f.Data_entry,
+    f.Validity,
+    f.CommentID,
+    CASE
+        WHEN EXISTS(
+        SELECT 'x'
+            FROM conflicts_unresolved cu
+        WHERE f.CommentID =cu.CommentId1 OR f.CommentID=cu.CommentId2
+    )
+    THEN 'Y'
+    ELSE 'N'
+    END AS Conflicts_Exist,
+    CASE ddef.Data_entry ='Complete'
+    WHEN 1 THEN 'Y'
+    WHEN null THEN 'Y'
+    ELSE 'N' 
+    END AS DDE_Complete
+QUERY;
+
+        $from = <<<QUERY
+FROM flag f
+    JOIN session s ON(s.ID=f.SessionID)
+    JOIN candidate c ON(c.CandID=s.CandID)
+    LEFT JOIN flag ddef ON(ddef.CommentID=CONCAT('DDE_', f.CommentID))
+QUERY;
+
+        $where = <<<QUERY
+WHERE f.CommentID NOT LIKE 'DDE%'
+    AND f.Test_name =:inst
+    AND s.Active    ='Y' AND c.Active='Y'";
+QUERY;
 
         if ($tablename === "") {
-            // the data is in the flag table, add the data column to the query and
-            // do not join the table.
+            // the data is in the flag table, add the data column to the query
+            // and do not join the table.
             $extraSelect = ", f.Data ";
 
             return $select . $extraSelect . $from . $where;
         }
 
         // add the SQL table to the query
         $extraSelect = ", i.* ";
-        $extraJoin = "JOIN " . $this->SQLDB->escape($tablename) . " i ON (i.CommentID=f.CommentID) ";
+        $extraJoin   = "JOIN " . $this->SQLDB->escape($tablename)
+        . " i ON(i.CommentID=f.CommentID) ";
```

Comment:
I'm surprised PHPCS allows this. It should be indented, not at the same level as the start of the statement.